lerna.json

```diff
@@ -0,0 +1,10 @@
+{
+  "lerna": "2.11.0",
```

Comment:
I don't know what that key in the config actually means. It must be there, otherwise lerna doesn't work. But the version is irrelevant since lerna is installed with the top-level package.json. Maybe it's for cases when you have lerna installed globally, and signals the compatibility?

Reply:
It's not in the lerna docs and for me it still works when I remove it.